client/components/nab-bar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default class Navbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      path: props.path
+    };
```

Comment:
Avoid storing things in state if you can. I think you can delete this `constructor`.